### DISCUSSION TITLE
Embedded Server

### DISCUSSION THREAD

**Discussion by cbrake** [nats-io/nats-server#3540] [General]
Looking at the roadmap, what is meant by the "Embedded Server (no TCP)" feature?

**Comment by kozlovic** [comment]
That you can configure the server to not start the TCP listen port and instead use that nats.go library to connect directly (memory/pipe connection). See test in client library: https://github.com/nats-io/nats.go/blob/543e628f4bafe989ce2ba548af09e2067c67ea69/nats_test.go#L2902
